api/caseshare/index.ts

```diff
@@ -1,7 +1,14 @@
+import {SharedCase} from '@hmcts/rpx-xui-common-lib/lib/models/case-share.model'
```

Comment:
common-lib is a frontend, while the model is just purely a js object think the preference would be for api to have it's own models